test/integration/chain/BtcdClient.spec.ts

```diff
@@ -11,18 +12,30 @@ describe('BtcdClient', () => {
   const testWif = 'cQ4crx5qPv7NDdj41ehumfB9f89zyWdggy8JnNDjKVQwsLswahd4';
   const testAddress = 'msRY4KpAJ8o9da1YEASy1j2ACnuzh4SyFs';
 
-  const client = new BtcdClient({
+  const client = new ChainClient({
     host: 'localhost',
     port: 18334,
-    user: 'user',
-    password: 'user',
+    rpcuser: 'user',
+    rpcpass: 'user',
     certpath: path.join('docker', 'btcd', 'data', 'rpc.cert'),
-  });
+  }, ChainType.BTC);
+
+  const ltcdClient = new ChainClient({
+    host: 'localhost',
+    port: 19334,
+    rpcpass: 'user',
+    rpcuser: 'user',
+    certpath: path.join('docker', 'ltcd', 'data', 'rpc.cert'),
+  }, ChainType.LTC);
 
-  it('should connect', async () => {
+  it('BtcdClient should connect', async () => {
     await expect(client.connect()).to.be.fulfilled;
   });
 
+  it('LtcdClient should connect', async () => {
+    await expect(ltcdClient.connect()).to.be.fulfilled;
```

Comment:
The Travis build is failing because your are not disconnecting from `ltcdClient`. Add something like `await ltcdClient.disconnect();` to the `after` method at the end of the file and you should be good.